clusterloader2/testing/density/config.yaml

```diff
@@ -0,0 +1,128 @@
+automanagedNamespaces: 3
+steps:
+- measurements:
+  - Identifier: TimerGlobal
+    Method: Timer
+    Params:
+      action: start
+  - Identifier: TimerCreate
+    Method: Timer
+    Params:
+      action: start
+  - Identifier: SchedulerLatency
```

Comment:
I would suggest removing this and adding later when the PR for SchedulerLatency is merged.

Reply:
done.